include/bluetooth/conn.h

```diff
@@ -309,6 +309,17 @@ enum {
 	BT_CONN_ROLE_PERIPHERAL = 1,
 };
 
+enum bt_conn_state {
+	/** Channel disconnected */
+	BT_CONN_STATE_DISCONNECTED,
+	/** Channel in connecting state */
+	BT_CONN_STATE_CONNECT,
```

Comment:
I would prefer to add "ING" -> BT_CONN_STATE_CONNECTING
- That is what the documentation says
- It ensures that the name is not a subset of the next name (CONNECTED).

Ditto for DISCONNECT -> DISCONNECTING

Reply:
I am not opposed to changing them either. If we decide to change them, so that change also be applied to the L2CAP, ISO and internal BT CONN states?

Reply:
Yes, it'd make sense to be consistent with the naming across the various layers and protocols

Reply:
Will add additional commit to fix that

Reply:
Was that additional commit ever added?
I would like to have it.  (But if we decided not to do that, I will approve as is.)

Reply:
It has not been added yet, will do that soon™

Reply:
Added it now. Will do the L2CAP renaming in a different PR as that requires an API change.

Reply:
https://github.com/zephyrproject-rtos/zephyr/pull/43165